extensions/big-data-cluster/src/bigDataCluster/constants.ts

```diff
@@ -25,6 +25,9 @@ export class IconPath {
 	public static sqlMasterNode: { dark: string, light: string };
 	public static copy: { dark: string, light: string };
 	public static refresh: { dark: string, light: string };
+	public static status_ok: { dark: string, light: string };
+	public static status_warning: { dark: string, light: string };
+	public static notebook: { dark: string, light: string };
```

Comment:
seems it is time to define an interface for icon :)

Reply:
Yeah I was thinking the same thing - laziness just got the best of me 😁